x-pack/plugins/session_view/public/components/session_view/session_view.tsx

```diff
@@ -4,44 +4,33 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
```

Comment:
Why do we want to change the file structure to this? 👀
If we decide to go with snake case, we will need to do the same for other components as well. I also see some other plugins have the structure of `/public/components/component_name/index.tsx` if we want to be consistent with them, but i dont think that's required 🤔

Reply:
Yea I think it's better if we keep it as `components/SessionView/Index.tsx` since that's what we've been using so far.

Reply:
Yea ok I see, paulo created a new index.tsx that re-exports this file. I'm personally not a fan of that (as mentioned in my PR review) haha

Reply:
Oh, I saw it under [STYLEGUIDE](https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#filenames), mentioning, that All filenames should use `snake_case`.

But, I now realized this line:

> This guide applies to all development within the Kibana project and is **recommended** for the development of all Kibana plugins.

So, since in the case of plugins, that is just a recommendation, think we are good to go with our current structure, I'll revert a few things